README.md

```diff
@@ -26,7 +26,7 @@ While comprehensive documentation remains a work in progress, our [quickstart]
 is well-polished and we refer users wishing to test drive Brigade 2 to
```

Comment:
I wonder if we want to make a few additional updates to this README, including (potentially):

 * Revising/rewording `While comprehensive documentation remains a work in progress` (Naturally, more docs are always helpful, but I think we're at a state where most docs are in and updated.
 * Revising/rewording `our quickstart is well-polished and we refer users wishing to test drive Brigade 2 to that documentation.` -- Do we drop '2' now that it is the de facto default?  Defer to your judgement here... I suppose it could also just stay the same/similar...
 * A link/mention to one or both of the v1 GH tree and v1 docs site now that they aren't the defaults

Reply:
For sure. I can include that in this PR. I think there will be some follow-ups to this PR as well if/when we identify more spots like this where we talk about v2 as something new/beta, under "heavy development," etc.

Reply:
I think I've applied all this feedback now. wdyt?